File: JavaScript/Day Seven/TaskEight.js
let sumOfMultiplesOfFour = 0;
let sumOfMultiplesOfEight = 0;

for(let count = 4; count <= 1024; count*= 4){
	sumOfMultiplesOfFour += count

}
for(let count = 8; count <= 32789; count*= 8){

	sumOfMultiplesOfEight += count

}
console.log(sumOfMultiplesOfFour +  sumOfMultiplesOfEight)

File: Python/Day Seven/task_eight.py
numberOne = 4;
numberTwo = 8;

sum_of_multiples_of_four = 0;
sum_of_multiples_of_eight = 0;

for num in range(1, 6):
	sum_of_multiples_of_four = sum_of_multiples_of_four + numberOne
	sum_of_multiples_of_eight = sum_of_multiples_of_eight + numberTwo

	

	numberOne = numberOne * 4;
	numberTwo = numberTwo * 8;
print(sum_of_multiples_of_four + sum_of_multiples_of_eight )